.github/gh-actions-self-hosted-runners/arc/environments/beam.env

```diff
@@ -31,11 +31,12 @@ deploy_webhook = "true"
 runner_group = "beam"
 main_runner = {
     name = "main-runner"
+    runner_image = "us-central1-docker.pkg.dev/apache-beam-testing/beam-github-actions/beam-arc-runner:738c02cac4d6a88faa3dd02606d50f1d2efed5e7"
     machine_type = "e2-standard-16"
     min_node_count = "1"
-    max_node_count = "7"
+    max_node_count = "10"
```

Comment:
Could we bump this up even further? For context, our Jenkins setup currently has 16 dedicated machines, each running 4 workers. So maybe jumping up to 16 makes sense?

Reply:
Sure. Ill bump it to 16 to be safe and as time goes by and more jobs are added we can do more precise numbers. 
They will scale down when not used so its fine

Reply:
Thanks!